internal/handlers/irc/handlers.go

```diff
@@ -23,21 +29,46 @@ func connectHandler(c Client) func(*girc.Client, girc.Event) {
 	}
 }
 
-func privMsgHandler(c Client) func(*girc.Client, girc.Event) {
+/*
+messageHandler handles the PRIVMSG IRC event, which entails both private
+and channel messages. However, it only cares about channel messages
+*/
+func messageHandler(c Client) func(*girc.Client, girc.Event) {
 	return func(gc *girc.Client, e girc.Event) {
-		formatted := fmt.Sprintf(c.MessageFormat, e.Source.Name, e.Params[1])
+		formatted := c.Settings.Prefix + e.Source.Name + c.Settings.Suffix + " " + e.Params[1]
```

Comment:
so this is fine, but we've actually historically made the `SUFFIX` variable set to `"> "` so we don't need to worry about the extra `+ " "` you have in there.